pages/1.11/deploying-services/marathon-on-marathon/advanced/index.md

```diff
@@ -357,288 +360,250 @@ In this step, a non-native Marathon instance is installed on DC/OS with the Meso
-      "user":"nobody",
-      "cpus":2,
-      "mem":4096,
-      "disk":0,
-      "instances":1,
-      "constraints":[  
-         [  
-            "hostname",
-            "UNIQUE"
-         ]
-      ],
-      "container":{  
-         "type":"DOCKER",
-         "docker":{  
-            "image":"<your-repo>/marathon-dcos-ee:<your-tag>",
-            "network":"HOST",
-            "privileged":false,
-            "parameters":[  
-
-            ],
-            "forcePullImage":false
-         },
-         "volumes":[  
-            {  
-               "containerPath":"/opt/mesosphere",
-               "hostPath":"/opt/mesosphere",
-               "mode":"RO"
-            }
-         ]
-      },
-      "env":{  
-         "JVM_OPTS":"-Xms256m -Xmx2g",
-         "DCOS_STRICT_SECURITY_ENABLED":"false",
-         "DCOS_SERVICE_ACCOUNT_CREDENTIAL_TOFILE":{  
-            "secret":"service-credential"
-         },
-         "MESOS_AUTHENTICATEE":"com_mesosphere_dcos_ClassicRPCAuthenticatee",
-         "MESOS_MODULES":"file:///opt/mesosphere/etc/mesos-scheduler-modules/dcos_authenticatee_module.json",
-         "MESOS_NATIVE_JAVA_LIBRARY":"/opt/mesosphere/lib/libmesos.so",
-         "MESOS_VERBOSE":"true",
-         "GLOG_v":"2",
-         "PLUGIN_ACS_URL":"http://master.mesos",
-         "PLUGIN_AUTHN_MODE":"dcos/jwt+anonymous",
-         "PLUGIN_FRAMEWORK_TYPE":"marathon"
-      },
-      "healthChecks":[  
-         {  
-            "path":"/ping",
-            "protocol":"HTTP",
-            "portIndex":0,
-            "gracePeriodSeconds":1800,
-            "intervalSeconds":10,
-            "timeoutSeconds":5,
-            "maxConsecutiveFailures":3,
-            "ignoreHttp1xx":false
-         }
-      ],
-      "secrets":{  
-         "service-credential":{  
-            "source":"<path-to-secret-name>"
-         }
-      },
-      "labels":{  
-         "DCOS_SERVICE_NAME":"<non-native-marathon>",
-         "DCOS_SERVICE_PORT_INDEX":"0",
-         "DCOS_SERVICE_SCHEME":"http"
+### Security Mode - Disabled
+
+```json
+{
+  "id": "/marathon-user-ee",
+  "cmd": "cd $MESOS_SANDBOX && LIBPROCESS_PORT=$PORT1 && /marathon/bin/start --default_accepted_resource_roles \"*\" --enable_features \"vips,task_killing,external_volumes,secrets,gpu_resources\" --framework_name marathon-user-ee --hostname $LIBPROCESS_IP --http_port $PORT0 --master zk://master.mesos:2181/mesos --max_instances_per_offer 1 --mesos_leader_ui_url /mesos --zk zk://master.mesos:2181/universe/marathon-user-ee",
+  "cpus": 2,
+  "mem": 2048,
+  "disk": 0,
+  "instances": 1,
+  "constraints": [
+    [
+      "hostname",
+      "UNIQUE"
+    ]
+  ],
+  "container": {
+    "type": "MESOS",
+    "docker": {
+      "image": "mesosphere/marathon-dcos-ee:__FILL_IN_VERSION_HERE__",
```

Comment:
`__FILL_IN_VERSION_HERE__` is something we might want to replace with a version that we recommend.

@zen-dog got a suggestion for us here (and everywhere else in this doc)?

Reply:
I actually need a recommendation here for being able to fully test everything - the reader might have the same problem.

Reply:
This is where you insert a specific image tag - which one do you want? We might have a dozen within each dc/os version and the recommendation is to take the "newest".

Reply:
@zen-dog the one we would recommended for 1.11 as that is what we are talking about in this document.
If there are specific recommendations for specific point-releases, then we should list them here.

Reply:
That's my point above - with DCOS 1.11 there might ~10 minor marathon releases and therefore 10 compatible images. The recommendation is to take the "latest currently available" but that might vary depending on specific bugs, customer problems etc.

Reply:
...but since we know which magor/minor Marathon versions belong to which dc/os version, we might be more specific with: `"image": "mesosphere/marathon-dcos-ee:v1.5._HIGHEST_AVAILABLE_PATCH_VERSION_"` or smth.